performance_manager/tests/test_postgres.py

```diff
@@ -30,21 +76,332 @@ def set_environment_variables() -> Iterator[None]:
                 if line.startswith("#") or line == "":
                     continue
                 key, value = line.split("=")
-                logging.info("setting %s to %s", key, value)
+                logging.debug("setting %s to %s", key, value)
                 os.environ[key] = value
 
+
+@pytest.fixture(scope="module", name="db_manager")
+def fixture_db_manager() -> DatabaseManager:
+    """
+    generate a database manager for all of our tests
+    """
+    set_env_vars()
+    db_manager = DatabaseManager()
+    return db_manager
+
+
+@pytest.fixture(autouse=True, name="s3_patch")
+def fixture_s3_patch(monkeypatch: MonkeyPatch) -> Iterator[None]:
```

Comment:
patch all of the s3 calls that the tests make. this way we don't have to change them in the codebase.